### PR TITLE
Laravel orchid installation and setup

### LINKED ISSUE #9: Setup Laravel Orchid for the Project
We need to integrate `Laravel Orchid` into our project to manage our application's backend functionality effectively. `Laravel Orchid` will provide us with a web-based, customizable administration panel to handle `CRUD` operations, `data-management` functionality, and other important backend tasks.

We will need to install `Laravel Orchid` using `Composer` and configure it within our project to take advantage of its features. Additionally, we will need to integrate it with our existing project structure and ensure that our `models`, `controllers`, and `views` are correctly configured to work with `Laravel Orchid`.

Once `Laravel Orchid` is fully integrated, we can use it to manage everything from user and role management to database table creation and management, saving us a significant amount of time and effort.

### PR DESCRIPTION
**Pull request by ExcelDurant**
This pull request sets up `Laravel Orchid` and adds instructions for setting up `Laravel Orchid` in the project.

closes #9

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -2,46 +2,70 @@
...
+        'name',
         'email',
         'password',
+        'permissions',
         'country',
     ];
 
     /**
-     * The attributes that should be hidden for serialization.
+     * The attributes excluded from the model's JSON form.
```
Comment (Fenn-CS) [inline app/Models/User.php:26]: Is is accurate?

File: app/Orchid/Layouts/Examples/ChartBarExample.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (Fenn-CS) [inline app/Orchid/Layouts/Examples/ChartBarExample.php:1]: This is still an example file. You should delete the whole example folder.

**Review by Fenn-CS — CHANGES_REQUESTED** [review]
Delete all example directories or files. We need only actual implementations that we use.

File: app/Models/User.php [hunk 3]
```diff
@@ -2,46 +2,70 @@
 
 namespace App\Models;
 
-// use Illuminate\Contracts\Auth\MustVerifyEmail;
```
Comment (Fenn-CS) [inline app/Models/User.php:5]: Why would remove all these user traits?

Comment (ExcelDurant) [reply app/Models/User.php:5]: The `Orchid` `Orchid\Platform\Models\User` model extends `Authenticatable`  which still has those traits

**Pull request merged**